Development.txt

```diff
@@ -1,3 +1,6 @@
 Hello this is main branch
 Recently added
 
+Kanagu is my name
```

Comment:
What is the purpose of this line of code ?

Reply:
It is your name